source/develop/release-management/features/storage/backup-storage-domain.html.md

```diff
@@ -0,0 +1,248 @@
+---
+title: BackupStorageDomain
+category: feature
+authors: mlipchuk, shubham0d
```

Comment:
@shubham0d you should add yourself to the `data/authors.yml` file too so your username renders nicely here.